service/src/main/java/org/apache/griffin/core/job/JobController.java

```diff
@@ -113,4 +113,10 @@ public JobHealth getHealthInfo() {
                 .contentType(MediaType.APPLICATION_OCTET_STREAM)
                 .body(resource);
     }
+
+    @RequestMapping(value = "/jobs/trigger/{id}", method = RequestMethod.GET)
+    @ResponseStatus(HttpStatus.NO_CONTENT)
```

Comment:
I think it makes sense to see how we can return jobinstance information from this method, so it would be possible to wait for job's status via api.